django/chat/management/commands/eval_responses.py

```diff
@@ -255,9 +255,9 @@ def _create_test_user():
 def _test_qa_response(eval_instance, user):
     try:
         # Create chat and build chat history
-        chat = Chat(user=user)
+        chat = Chat.objects.create(user = user, mode = eval_instance["mode"])
```

Comment:
The original version had `chat = Chat(user=user)`. That bypassed the custom create method in ChatManager, so the resulting object had no security label which broke things upstream.

Reply:
Great, thanks. There shouldn't be spaces around the keyword arguments = though. Can you make sure your format-on-save is working?